lib/PearlBee/Comments.pm

```diff
@@ -0,0 +1,29 @@
+package PearlBee::Comments;
+use Dancer2 appname => 'PearlBee';
+use Dancer2::Core;
+use Class::Load qw(load_class load_optional_class);
+use Carp qw(croak);
+
+my $name = config->{comments} || 'builtin';
+my $camelized = Dancer2::Core::camelize($name);
+my $component_class = "PearlBee::Comments::$camelized";
+
+load_class($component_class);
+load_optional_class("${component_class}::Endpoints");
```

Comment:
I don't like this name... Does anybody have any suggestions?
The goal is just to separate `$component_class` (e.g. `PearlBee::Comments::Builtin`, which will be instantiated, and has a few attributes) from Dancer DSL present in `PearlBee::Comments::Builtin::AddComment` or `PearlBee::Comments::Builtin::Dashboard` for example. I want to be able to test the main class without loading Dancer.